tests/Paket.Tests/InstallModel/Xml/FrameworkSpecificFiles.fs

```diff
@@ -0,0 +1,97 @@
+﻿module Paket.InstallModel.Xml.FrameworkSpecificFilesSpecs
+
+open Paket
+open NUnit.Framework
+open FsUnit
+open Paket.TestHelpers
+open Paket.Domain
+open Paket.Requirements
+
+let modelFromFiles files =
+    
+    InstallModel.CreateFromLibs(PackageName "FrameworkSpecificFiles", SemVer.Parse "0.21", FrameworkRestriction.NoRestriction,
+        []
+        |> Paket.InstallModel.ProcessingSpecs.fromLegacyList @"..\FrameworkSpecificFiles\",
+        files
+        |> Paket.InstallModel.ProcessingSpecs.fromLegacyList @"..\FrameworkSpecificFiles\",
+        [],
+            Nuspec.All)
+
+[<Test>]
+let ``https://github.com/fsprojects/Paket/issues/2392``() =
+    
+    // we have two folders, but it should only ever take one
+    let files = [ @"..\FrameworkSpecificFiles\build\net40\FrameworkSpecificFiles.props"
+                  @"..\FrameworkSpecificFiles\build\FrameworkSpecificFiles.props"
+    ]
+
+    let expectedPropertyDefinitionNodes = """<?xml version="1.0" encoding="utf-16"?>
+<Choose xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
> The Choose element contains a series of When elements with Condition attributes that are tested in order __from top to bottom__ until one evaluates to true. If more than one When element evaluates to true, only __the first one__ is used.

https://msdn.microsoft.com/en-us/library/ms164307.aspx

This means the first group is always chosen, instead of the more specific one.

Not sure yet if it has _always been like this_, or actually introduced through my changes.